xtask/src/build_docker_image.rs

```diff
@@ -363,6 +363,7 @@ fn has_native_image(
     Ok(match target.sub.as_deref() {
         // FIXME: add additional subs for new Linux distros, such as alpine.
         None | Some("centos") => match (platform, target.name.as_str()) {
+            // TODO: "windows/amd64"?
```

Comment:
not needed, we will never have a msvc image available like this through a "native" extra dockerfile